src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[] | []
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.length && movies.map((movie) => (
+      <MovieCard movie={movie} key={movie.imdbId} />
+    ))}
+  </div>
```

Comment:
you shouldn't have change semantic tags for divs
![image](https://user-images.githubusercontent.com/59209337/221116118-6b6a2223-8b17-467e-bd56-fb650b7addb0.png)